jumpscale/packages/vdc_dashboard/bottle/vdc_helpers.py

```diff
@@ -1,10 +1,32 @@
 from bottle import HTTPResponse, abort
-import math
 from jumpscale.loader import j
 
 from jumpscale.sals.vdc.size import VDC_SIZE
 from jumpscale.sals.vdc.models import KubernetesRole
 
+CURRENTVDC = None
+
+
+def require_vdc_info(fun):
+    def inner(*args, **kwargs):
+        global CURRENTVDC
+        if not CURRENTVDC:
+            CURRENTVDC = get_vdc()
+        return fun(*args, **kwargs)
```

Comment:
we can pass the vdc here too as the first argument, instead of referencing the global variable of `CURRENTVDC`. Just like `vdc_route`.